sigar_interface.go

```diff
@@ -353,19 +353,26 @@ type ProcIo struct {
 }
 
 type ProcMem struct {
-	Size        uint64
-	Resident    uint64
-	Share       uint64
-	MinorFaults uint64
-	MajorFaults uint64
-	PageFaults  uint64
+	Size          uint64
+	Resident      uint64
+	Share         uint64
+	MinorFaults   uint64
+	MajorFaults   uint64
+	PageFaults    uint64
+	PageFileBytes uint64
 }
 
 type ProcTime struct {
-	StartTime uint64
-	User      uint64
-	Sys       uint64
-	Total     uint64
+	CollectionTime time.Time
+	StartTime      uint64
+	User           uint64
+	Sys            uint64
+	Total          uint64
```

Comment:
nit: Maybe comment what time unit these `uint64`s are for.  I think it's millis?